printing/paper

```diff
@@ -96,7 +102,10 @@ def main(argv=None):
         print(bold(red("The user {} doesn't exist.".format(args.user))))
         return 1
 
-    return commands[args.command](args)
+    if args.command == 'refund' or args.command == 'forward':
```

Comment:
Instead of an if statement here, which makes the subcommand logic harder to understand, you should put the same if statement in the `adjust` command (which has access to `args` and therefore `args.command`).